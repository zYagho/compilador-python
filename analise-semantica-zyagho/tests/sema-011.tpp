{Aviso: Coerção implícita do valor de 'b', 'inteiro' para 'flutuante'}
{Aviso: Coerção implícita do valor de 'a', 'flutuante' para 'inteiro'}
{Erro: Função 'principal' deveria retornar 'inteiro', mas retorna 'vazio'}

flutuante: a
inteiro: b

inteiro principal()
	b := 18
	a := 1.0

	a := b
	b := a
fim
