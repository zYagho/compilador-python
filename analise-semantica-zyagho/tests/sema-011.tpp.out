WAR-SEM-IMP-COERC-OF-VAR
WAR-SEM-IMP-COERC-OF-VAR
ERR-SEM-FUNC-RET-TYPE-ERROR
